app/partials/analysis/tools/powercalc/main.jade

```diff
@@ -473,14 +473,18 @@ div(ng-controller="powercalcMainCtrl as mainArea")
         div(style='margin-top:20px;')
           div(style='display:inline')
             | Power: &nbsp
-            input#twoTestpower_v(ng-disabled="mainArea.deployed" type="number" step="any" name='twoTestpower',style='border:0; background-color:#f5f5f5; color:#f6931f; font-weight:bold; display:inline;' ng-click="mainArea.twoTestSync()" ng-keypress="mainArea.twoTestPress($event)")
+            input#twoTestpower_v(ng-disabled="mainArea.deployed" type="number" step="any" name='twoTestpower',style='border:0; background-color:#f5f5f5; color:#f6931f; font-weight:bold; display:inline;' ng-click="mainArea.twoTestPower()" ng-keypress="mainArea.twoTestPress($event)")
             #twoTestpowerui
         button.btn.btn-primary.btn-sm(
           ng-hide="mainArea.deployed"
           type="submit"
           aria-label="Reset"
-          ng-click="mainArea.twoTestReset()")
+          ng-click="mainArea.reset()")
```

Comment:
actually, previous function name `twoTestReset` was more meaningful, since it specifically related to the twoTest calculator, while `mainArea.reset` sounds like the function that completely resets everything in the main controller. also, your other functions for this calculator start with `twoTest..`, so you should be consistent in naming and either rename them as well, or rename this one to start with `twoTest..`. remember, other people will read and try to understand your code.